WindowsServerDocs/remote/remote-access/directaccess/DirectAccess-Known-Issues.md

```diff
@@ -10,8 +10,19 @@ ms.date: 08/07/2020
 ---
 # DirectAccess Known Issues
 
->Applies To: Windows Server (Semi-Annual Channel), Windows Server 2016
+>Applies To: Windows Server (Semi-Annual Channel), Windows Server 2016, Windows Server 2019
 
+## DNS registration of DirectAccess client IPv6 addresses
+
+Starting with the Windows 10 May 2020 Update, a client no longer registers its IP addresses on DNS servers configured in a Name Resolution Policy (NRPT).
+If the DNS registration is needed, for e.g. Manage Out, it can be explicitely enabled with this registry key on the client:
+
+Path: ```HKLM\System\CurrentControlSet\Services\Dnscache\Parameters```   
+Type: ``DWORD``   
+Value name: ``DisableNRPTForAdapterRegistration``   
+Values:   
+``1`` - DNS Registration disabled (default since Windows 10 May 2020 Update)   
+``0`` - DNS Registration enabled
```

Comment:
(ref. your other PRs #5207 & #5208 covering the same topic)
```suggestion
Starting with the Windows 10 May 2020 Update, a client no longer registers its IP addresses on DNS servers configured in a Name Resolution Policy Table (NRPT).
If DNS registration is needed, for example **Manage Out**, it can be explicitly enabled with this registry key on the client:

Path: `HKLM\System\CurrentControlSet\Services\Dnscache\Parameters`
Type: `DWORD`
Value name: `DisableNRPTForAdapterRegistration`
Values:
`1` - DNS Registration disabled (default since the Windows 10 May 2020 Update)
`0` - DNS Registration enabled
```